plugins/grid-bookmark/src/GridBookmarkWidget/components/GridBookmarkWidget.tsx

```diff
@@ -15,15 +15,11 @@ import { GridBookmarkModel } from '../model'
 
 const useStyles = makeStyles()({
   card: {
-    display: 'flex',
-    flexFlow: 'column',
-    margin: '5px',
-    padding: '5px',
-    gap: '5px',
+    marginTop: 5,
```

Comment:
removed the margin and padding, to give slighly more space to data grid. the flex display seemed like it could be removed also